cmd/antrea-agent/agent.go

```diff
@@ -294,6 +294,16 @@ func run(o *Options) error {
 		NodePortAddressesIPv6: nodePortAddressesIPv6,
 	}
 
+	if features.DefaultFeatureGate.Enabled(features.SecondaryNetwork) {
+		bridgeConfig := o.config.SecondaryNetwork.OVSBridges[0]
+		ovsSecBridgeClient := ovsconfig.NewOVSBridge(bridgeConfig.BridgeName, ovsDatapathType, ovsdbConnection, ovsconfig.WithRequiredPortExternalIDs(interfacestore.AntreaInterfaceTypeKey))
+		secondaryBridge :=agent.NewSecondaryBridgeInitializer(ovsSecBridgeClient, ifaceStore, bridgeConfig.BridgeName)
```

Comment:
As far as I can tell there is already a different interface store for secondary network interfaces in the `PodController`: https://github.com/antrea-io/antrea/blob/325890854da4fe0d5168bc7f516f84d4ef85eb35/pkg/agent/secondarynetwork/podwatch/controller.go#L92

So it doesn't seem right to use the "primary" one here? It feels like the primary one will not be updated when a new secondary network interface is created for a Pod.

I would expect that the scenario described in https://github.com/antrea-io/antrea/issues/6578 would still fail  even with this patch, as when deleting the Pod we would not find the interface in the "correct" interface store?

Maybe @jianjuns can comment as to why we have a separate interface store for secondary network interfaces, but it makes sense to me.

Reply:
The same as the OVS bridgeClient, a new `ovsBridgeClient` is created in function secondarynetwork.NewController(). it would be better to move the initialize functions to `secondarynetwok.Controller` rather than introducing a new initializer object?

Reply:
Sure, I will update the code.